automation/Dockerfile.fedora

```diff
@@ -15,7 +15,9 @@ RUN dnf -y install --setopt=install_weak_deps=False \
                    rpm-build git python3-devel python3-dbus python3-coveralls \
                    python3-pyyaml python3-jsonschema python3-setuptools \
                    NetworkManager-ovs openvswitch libibverbs python36 \
-                   python3-gobject-base dnsmasq radvd python3-tox python2 && \
+                   python3-gobject-base dnsmasq radvd python3-tox python2 \
```

Comment:
Since there are so many dependencies, I would prefer to have only one per line and grouped by why they are there to make it easier to review them.

Reply:
Done. Add a comment:

`Below packages for pip(used by tox) to build python-gobject`

Reply:
Strictly speaking, there is a space missing before the bracket: `pip (used for tox)`.